network.go

```diff
@@ -61,6 +61,7 @@ type NetworkInfo interface {
 	IpamConfig() (string, []*IpamConf, []*IpamConf)
 	IpamInfo() ([]*IpamInfo, []*IpamInfo)
 	DriverOptions() map[string]string
+	NetworkOptions() map[string]string
```

Comment:
Libnetwork options are well-known options being set via NetworkOption setter functions and not via a map[string]string like opaque driver options.
Also, as I commented on the docker PR, we cannot assume future network option can nicely fit in a  string value. 

I'd rather have an explicit structure containing the network options or a getter function for each of the options. Regarding this PR, something like
`Internal() bool`